Dockerfile

```
@@ -15,33 +15,21 @@ LABEL "maintainer"="https://github.com/laminas/technical-steering-committee/"
 
 WORKDIR /app
 
-RUN apt update \
-    && apt install -y software-properties-common \
-    && add-apt-repository -y ppa:ondrej/php \
-    && apt install -y \
-        git \
-        gnupg \
-        libzip-dev \
-        zip \
-        php8.0-cli \
-        php8.0-curl \
-        php8.0-mbstring \
-        php8.0-readline \
-        php8.0-xml \
-        php8.0-zip \
-    && apt clean
-
-ADD composer.json /app/composer.json
-ADD composer.lock /app/composer.lock
+RUN apk add --no-cache git gnupg libzip \
+    && apk add --no-cache --virtual .build-deps libzip-dev \
+    && docker-php-ext-install zip \
+    && apk del .build-deps
+
+COPY composer.* /app/
 
 RUN COMPOSER_CACHE_DIR=/dev/null composer install --no-dev --no-autoloader
 
 # @TODO https://github.com/laminas/automatic-releases/issues/8 we skip `.git` for now, as it isn't available in the build environment
 # @TODO https://github.com/laminas/automatic-releases/issues/9 we skip `.git` for now, as it isn't available in the build environment
 #ADD .git /app/.git
-ADD bin /app/bin
-ADD src /app/src
+COPY bin /app/bin/
+COPY src /app/src/
 
-RUN composer install -a --no-dev
+RUN composer dumpautoload -a --no-dev
```

```suggestion
RUN composer dump-autoload -a --no-dev
```

See https://github.com/composer/composer/blob/e3af4ee606023079d8af222da82a5b6f36ca879f/src/Composer/Command/DumpAutoloadCommand.php#L29-L30